testsmanual/BenchmarkDotNet.IntegrationTests.CsProjNetFramework/MultipleFrameworksTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Tests.XUnit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace BenchmarkDotNet.IntegrationTests
+{
+    public class MultipleFrameworksTest : BenchmarkTestExecutor
+    {
+        private const string Prefix = "// ### Called: ";
+        private const string GlobalSetupCalled = Prefix + "GlobalSetup";
+        private const string GlobalCleanupCalled = Prefix + "GlobalCleanup";
+        private const string IterationSetupCalled = Prefix + "IterationSetup";
+        private const string IterationCleanupCalled = Prefix + "IterationCleanup";
+
+        private const string ExpectedBenchmarkNet461 = Prefix + "Benchmark NET461";
+        private const string ExpectedBenchmarkNet48 = Prefix + "Benchmark NET48";
+        private const string ExpectedBenchmarkNetCoreApp20 = Prefix + "Benchmark NETCOREAPP2_0";
+        private const string ExpectedBenchmarkNet70 = Prefix + "Benchmark NET7_0";
+#if NET461
+        private const string BenchmarkCalled = ExpectedBenchmarkNet461;
+#elif NET48
+        private const string BenchmarkCalled = ExpectedBenchmarkNet48;
+#elif NETCOREAPP2_0
+        private const string BenchmarkCalled = ExpectedBenchmarkNetCoreApp20;
+#elif NET7_0
+        private const string BenchmarkCalled = ExpectedBenchmarkNet70;
+#endif
+
+        public MultipleFrameworksTest(ITestOutputHelper output) : base(output) { }
+
+        private static string[] GetActualLogLines(Summary summary)
+            => GetSingleStandardOutput(summary).Where(line => line.StartsWith(Prefix)).ToArray();
+
+        [Theory]
+        [InlineData(RuntimeMoniker.Net461)]
+        [InlineData(RuntimeMoniker.Net48)]
+        [InlineData(RuntimeMoniker.NetCoreApp20)]
```

Comment:
our CI has no .NET Core 2.0 SDK installed, it's not going to work.

Reply:
No problem, the tests are manual only.